docs_2.0/manual_EN/4.ngql-guide/16.query-tuning-statements/1.explain-and-profile.md

```diff
@@ -0,0 +1,112 @@
+# EXPLAIN and PROFILE
+
+`EXPLAIN` helps output the execution plan of an nGQL statement without executing the statement. `PROFILE` executes the statement, then output the execution plan as well as the execution profile. You can optimize the queries for better performance with the execution plan and profile.
+
+## Execution Plan
+
+The execution plan is determined by the execution planner in the Nebula Graph query engine.
+
+Execution planner processes the parsed nGQL statements into actions. An action is the smallest unit that can be executed. A typical action could be fetching all neighbors of a given vertex, getting the properties of an edge, or filtering vertices or edges based on the given condition. Each action is assigned to an operator which performs the action.
```

Comment:
the given condition.==> "the given conditions."  OR "a given condition."

Reply:
which performs the action. ==> that performs the action.

THAT is preferred to introduce an essential adjective clause.

Reply:
Thanks and fixed.